scripts/inventories/group_vars/jenkins/credentials.yml

```diff
@@ -46,3 +46,12 @@ jenkins_secret_text_credentials:
       id: ""                                   # AWS client ID
       secret: ""                               # AWS secret token
       description: "AWS Client ID/Token"
+  - qe-gitlab-access-token:
+      secret: ""
+      description: "GitLab token used by bot to push to GL repo"
+  - qe-iam-access-key-id:
+      secret: ""
+      description: "aws_access_key_id"
+  - qe-iam-access-secret-key:
+      secret: ""
+      description: "aws_secret_access_key"
```

Comment:
Do we need to prepend "qe" to these names? Seems like they are fairly generic and we should just be asking for an "aws-credentials" name.

Reply:
we probably don't